app/_src/guides/consumer-producer-policies.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Namespace scoped policies
```

Comment:
I don't have strong opinion here, but should the title mention "producer" and "consumer"? like "Producer and Consumer Policies"?

Reply:
changed